File: OLSource1/v1/VS_officedev/codesnippet/CSharp/walkthrough--binding-to-data-from-a-service-in-a-vsto-add-in-project_3.cs
        private void InitializeServiceObjects()
        {
            request = new ContentService.getContentRequest();
            proxy = new ContentService.ContentServicePortTypeClient();
            document = new ContentService.requestedDocument[1];
            response = new ContentService.getContentResponse();
            appId = new ContentService.appId();
            components = new System.ComponentModel.Container();
            primaryDocumentsBindingSource = new System.Windows.Forms.BindingSource(this.components);
        }

File: OLSource1/v1/VS_officedev/codesnippet/VisualBasic/walkthrough--binding-to-data-from-a-service-in-a-vsto-add-in-project_3.vb
    Private Sub InitializeServiceObjects()
        request = New ContentService.getContentRequest()
        proxy = New ContentService.ContentServicePortTypeClient()
        document = New ContentService.requestedDocument(0) {}
        response = New ContentService.getContentResponse()
        appId = New ContentService.appId()
        components = New System.ComponentModel.Container()
        primaryDocumentsBindingSource = New System.Windows.Forms.BindingSource(components)
    End Sub